### ISSUE TITLE
[meta] Simplify tokenserver

### ISSUE THREAD

**Issue by pjenvey** [mozilla-services/tokenserver-rs#9]
A meta issue to track what potential improvements can be made to tokenserver's design during the rewrite to rust.

- Removal of multi-service support 
  - [Assignment](https://github.com/mozilla-services/tokenserver-rs/issues/2) methods require a "service" (stored in the db) for each call. The original intention was to support services other than service = `sync-1.5` but this was never used. We should be able to remove all of this and have tokenserver responsible for solely one service.

**Comment by pjenvey** [comment]
Over to @rfk - does the multi-service removal sound correct? Please add any additional ideas that come up here (and feel free to edit my first post)

**Comment by rfk** [comment]
> does the multi-service removal sound correct?

Yes, spot on. Relatedly, the [`services` table](https://github.com/mozilla-services/tokenserver/blob/master/tokenserver/assignment/sqlnode/schemas.py#L83) has some ability to store a custom URL pattern for the service, which we can probably also get rid of.

The other thing that may be worth cleaning up, is the way that we internally identify users via an "email" column. This is not actually their email address! It's a hangover from browserid and in practice is always a concatenation of their FxA userid and the FxA "issuer" domain, as seen e.g. [here](https://github.com/mozilla-services/tokenserver/blob/f19ac0e8402b8203c1db51e9321ea50cb361f634/tokenserver/verifiers.py#L237) where we reconstruct it from the info returned by the OAuth verifier.  I think it is important to continue to identify users by the unique tuple `(userid, issuer)` since it's useful in some configurations where we want to support users from e.g. both Prod and Stage FxA. But perhaps we could store it in a less confusing way than calling it "email".

**Comment by rfk** [comment]
Unassigning this and bumping it back to you @pjenvey :-)
